src/components/Admin/Messenger.jsx

```diff
@@ -7,90 +7,60 @@ import Input from "./Input";
 import Textarea from "./Textarea";
 import Button from "./Button";
 import toast from "react-hot-toast";
-
-const ADDRESSEES = [
-  {
-    uid: "1",
-    name: "Menthy Wu",
-    email: "yhung022@ucr.edu",
-    team: "b",
-    major: "Computer Science",
-    status: "pending",
-    selected: false,
-  },
-  {
-    uid: "2",
-    name: "Divyank Shah",
-    email: "yhung022@ucr.edu",
-    team: "c",
-    major: "Computer Science",
-    status: "accepted",
-    selected: false,
-  },
-  {
-    uid: "3",
-    name: "Shing Hung",
-    email: "yhung022@ucr.edu",
-    team: "d",
-    major: "Computer Science",
-    status: "rejected",
-    selected: false,
-  },
-];
+import Upload from "./Upload";
+import { messengerFilters } from "@/data/Filters";
 
 const Messenger = () => {
-  const [subjectText, setSubjectText] = useState("");
-  const [messageBody, setMessageBody] = useState("");
-  const [filters, setFilters] = useState({
-    accepted: false,
-    rejected: false,
-    pending: false,
-    volunteers: false,
-    mentors: false,
-    judges: false,
+  const [emails, setEmails] = useState({
+    sendto: [],
+    subject: "",
+    body: "",
+    files: [],
   });
-  const [filteredParticipants, setfilteredParticipants] = useState(ADDRESSEES);
+  const [filters, setFilters] = useState(messengerFilters);
   const clickHandler = () => {
-    if (subjectText === "") {
+    if (emails.subject === "") {
       toast("❌ Please add a subject!");
       return;
     }
-    if (messageBody === "") {
+    if (emails.body === "") {
       toast("❌ Please add a body!");
       return;
     }
-    const emails = filteredParticipants.map((user) => user.email);
-    console.log({
-      sendto: emails,
-      subject: subjectText,
-      body: messageBody,
-    });
-    toast(`✅ Email sent successfully!`);
+    toast(`✅ Emails sent successfully!`);
+    console.log(emails);
   };
   return (
-    <div className="w-full font-poppins h-full">
-      <div className="flex flex-col pb-3 pt-4 h-full items-stretch">
+    <div className="w-full font-poppins h-full flex flex-col justify-between">
+      <div className="flex flex-col pb-3 pt-4 h-full items-stretch justify-between">
         <Title title="Messenger" />
         <div className="flex items-center my-1">
           <p className="text-lg font-extrabold mr-5 my-0">to:</p>
           <Filters
             filters={filters}
             setFilters={setFilters}
-            setObjects={setfilteredParticipants}
-            objects={ADDRESSEES}
+            setObjects={() => {}}
```

Comment:
is it possible to not even mention these params, and instead have them as defaults in the function definition